tensorboard/BUILD

```diff
@@ -146,14 +183,6 @@ py_library(
     visibility = ["//visibility:public"],
 )
 
-py_library(
-    name = "expect_tensorflow_installed",
```

Comment:
Let's actually leave this rule for now and make it a dependency of `//tensorboard/compat:tensorflow` with a comment explaining that it's not actually a strict dependency any more (and obviously in open-source it never had a functional effect), but it's there so that tools can replace it with a real tensorflow dependency.  This is what we do within Google, so that will help ensure our internal build doesn't break.